.gitignore

```diff
@@ -1,6 +1,7 @@
 # ignore the virtualenv folder
 test_emails/
 venv
+myenv
```

Comment:
The commit is still polluted with the virtual environment file. Fix it before making the PR. Closing this one